egs/mgb2/asr/local/mgb_train_lms_extra_pocolm.sh

```diff
@@ -0,0 +1,160 @@
+#!/bin/bash
```

Comment:
We do not have to include this.
Please remove it.

Reply:
done